app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -1064,7 +1064,7 @@ private void setFullscreen(boolean enabled) {
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
                 getWindow().setDecorFitsSystemWindows(false);
                 if (getWindow().getInsetsController() != null) {
-                    getWindow().getInsetsController().hide(WindowInsets.Type.statusBars() | WindowInsets.Type.navigationBars());
```

Comment:
Why is this removed?

Reply:
It is due to a weird bug that occurs when exiting fullscreen mode of the LoyaltyCardViewActivity, where if you are using the 3-button navbar, all the buttons would be white (which would essentially become invisible if you set the theme to light mode, which makes the navbar background white), as opposed to gray. Not hiding the navbar during this activity would prevent that bug from happening (and I also don't think hiding the navbar was necessary).